app/models/comment.rb

```diff
@@ -41,6 +41,27 @@ class Comment < ActiveRecord::Base
 
   define_counter_cache(:versions_count) { |comment| comment.versions.count }
 
+  def parent_event
+    if parent_id # comment is a reply
+      first_ancestor.created_event
+    else
+      discussion.created_event
+    end
+  end
+
+  def first_ancestor
```

Comment:
arr! I thought you missed this. This is the comment parent. not the event parent. So if you reply to a reply, the comment tree structure is accurate, and may be many levels deep, but the event parent structure is formed around display so to do this, we find the first comment ancestor for the event parent.... but this way there is still an accurate structure within the comment tree. make sense?

Reply:
I thought we were doing a similar-to-facebook thing where instead of replying to a reply, we put another comment in the same strand, including a mention to the previous comment's author? Do you feel like we need a full tree structure for comments, especially if we're not displaying them that way?